spec/features/shelters/user_can_see_one_shelter_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe "show shelter by id page", type: :feature do 
+  it "can see all shelter's name and address" do
+    shelter_1 = Shelter.create(id: 1,
+                               # is adding the id num above what I should be doing? 
+                               name: "Rocky Mountain Puppy Rescue",
+                               address: "10021 E Iliff Ave",
+                               city: "Aurora",
+                               state: "CO",
+                               zip: "80247")
+
+    visit "/shelters/1"
+
+    expect(page).to have_content(shelter_1.name)
+    expect(page).to have_content(shelter_1.address)
+    expect(page).to have_content(shelter_1.city)
+    expect(page).to have_content(shelter_1.state)
+    expect(page).to have_content(shelter_1.zip)
+  end
+
+  it "can link back to shelter index" do
+    shelter_1 = Shelter.create(id: 1,
```

Comment:
Is manually adding an id here for a record what I should be doing? Or is there a way to access the id that lives in the test db?